backend/core/src/listings/listings-csv-exporter.service.ts

```diff
@@ -31,12 +31,12 @@ export class ListingsCsvExporterService {
     const listingObj = listings.map((listing) => {
       return {
         ID: listing.id,
-        "Created At Date (UTC)": formatDate(listing.createdAt, "MM-DD-YYYY hh:mm:ssA"),
+        "Created At Date": formatLocalDate(listing.createdAt, "MM-DD-YYYY hh:mm:ssA z", timeZone),
```

Comment:
It might be beneficial to create a backend test for this `exportFromObject` function. Or at minimum have tests for the `formatLocalDate`. Right now our front-end tests just validates that the csv is downloaded, but not what is in it which is something we can test from the backend